CorosHeader header
int32 message_code
int32 agent_id
string delivery_status

